### PR TITLE
Bless Pool Players through play button

### PR DESCRIPTION
**Pull request by awaismbajwa**
REDMINE-17702

### CODE REVIEW

File: entry_types/scrolled/package/src/frontend/PlayerControls/PlayPauseButton.js [hunk 1]
```diff
@@ -29,6 +32,10 @@ function clickHandler(props) {
       props.pause();
     } else {
       props.play();
+      if (!isBlessed) {
+        media.mute(false);
+        isBlessed = true;
+      }
     }
```
Comment (tf) [inline entry_types/scrolled/package/src/frontend/PlayerControls/PlayPauseButton.js:40]: `PlayerControls` is a presentational/dumb component that so far does not know about concepts like media. There is also `MediaPlayerControls`, which wires up the `PlayerControls` with `playerState`/`playerActions`. I think, it would be a better fit for media related functionality. Can you move this to a separate PR so it can already be merged?

Comment (awaismbajwa) [reply entry_types/scrolled/package/src/frontend/PlayerControls/PlayPauseButton.js]: I thought this player actions.play might get invoked from other places. I have rebased this PR can be merged.
